python/src/aiconfig/scripts/rage/rage.py

```diff
@@ -0,0 +1,60 @@
+import os
+import lastmile_utils.lib.core.api as core_utils
+import sys
+import time
+import itertools
+import numpy as np
+from result import Ok, Result
+
+logger = core_utils.get_logger(__name__)
+
+
+class RageConfig(core_utils.Record):
+    log_level: str | int = "WARNING"
```

Comment:
Do we need this? I don't see it being used

Or is it an abstract field we need to override for `core_utils.Record`?

Reply:
I'm going to use this in a subsequent PR. It's nice to have every subcommand have a controllable logging level.

Reply:
Can we move it to subsequent PR then until it's needed? Keeps things easier to logic together and review

Reply:
If you want we can keep the line just commented out

Reply:
Wait, I am using it